<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>341</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QStackedWidget" name="frames">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>231</width>
      <height>221</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>7</number>
    </property>
    <widget class="QWidget" name="circle">
     <widget class="QDoubleSpinBox" name="radiusSpinBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <double>50.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="triangle">
     <widget class="QDoubleSpinBox" name="side1trianglespin">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="value">
       <double>50.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="side2trianglespin">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="value">
       <double>50.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="side3trianglespin">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="value">
       <double>50.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="text_label6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Side 1:</string>
      </property>
     </widget>
     <widget class="QLabel" name="text_label5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Side 2:</string>
      </property>
     </widget>
     <widget class="QLabel" name="text_label4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Side 3:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="rectangle">
     <widget class="QDoubleSpinBox" name="widthrectspinbox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="value">
       <double>100.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="heightrectspinbox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="value">
       <double>50.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Height:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Width:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="star">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of apexes:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="apexes_spinbox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>5</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="radius1_spinbox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="value">
       <double>40.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="radius2_spinbox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="value">
       <double>80.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Radius1:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Radius2:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Right_polygone">
     <widget class="QSpinBox" name="spinboxnumapexes_rp">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>4</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinboxside_rd">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="value">
       <double>50.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="rhombus">
     <widget class="QDoubleSpinBox" name="spinbox_side_rh">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="value">
       <double>100.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spinbox_angle_rh">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>179.000000000000000</double>
      </property>
      <property name="value">
       <double>60.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Side:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Angle:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Side:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="value">
       <double>50.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="custom">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Change custom</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>382</x>
      <y>20</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>circle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>triangle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>rectangle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Star</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Right Polygone</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rhombus</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Square</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Custom</string>
     </property>
    </item>
   </widget>
   <widget class="QFrame" name="rotate_frame">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>360</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QDoubleSpinBox" name="rotationcenteryspinbox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="rotationcenterxspinbox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotation center coordinates:</string>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect center with shape</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="Changeframe">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>511</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QDoubleSpinBox" name="cahgexspinbox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>0</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>0</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="changeyspinbox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="text_label1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>scale:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="scalespinbox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="text_label2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>area:</string>
     </property>
    </widget>
    <widget class="QLabel" name="area_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="text_label3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>perimeter:</string>
     </property>
    </widget>
    <widget class="QLabel" name="perimeter_label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>0</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotate angle:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="rotationspinbox">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>0</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
